drawing_agent/app/nodes.py

```
@@ -11,267 +12,262 @@
-    if width_match:
-        dimensions["width_mm"] = float(width_match.group(1))
-    height_match = re.search(r'Высота:\s+([\d\.]+)\s+мм', result)
-    if height_match:
-        dimensions["height_mm"] = float(height_match.group(1))
-    lines_match = re.search(r'Количество линий:\s+(\d+)', result)
-    if lines_match:
-        dimensions["lines_count"] = int(lines_match.group(1))
+
+    w = re.search(r'Ширина:\s+([\d\.]+)\s+мм', result)
+    h = re.search(r'Высота:\s+([\d\.]+)\s+мм', result)
+    l = re.search(r'Количество линий:\s+(\d+)', result)
```

Ruff (0.15.12)

[error] Ambiguous variable name: `l`

(E741)